packages/inertia/src/inertia.js

```diff
@@ -154,7 +160,7 @@ export default {
             })
           }
 
-          this.saveScrollPositions()
+          scrollRegions.forEach(region => this.addScrollListener(region))
```

Comment:
This would apply the 25ms 'workaround' that prevents the scroll position to be saved instantly on page transition:

```suggestion
          setTimeout(() => scrollRegions.forEach(region => this.addScrollListener(region)), 25)
```